### PR TITLE
[master] Format markdown

### PR DESCRIPTION
**Pull request by mattmoor**
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`
/assign helmick
/cc helmick

### CODE REVIEW

File: docs/deploying.md [hunk 1]
```diff
@@ -81,13 +83,13 @@ order of operations is:
 Build new services by using the script at `./scripts/build`, specifying the
 following values:
 
--   `PROJECT_ID` (required) - your Google Cloud project ID.
```
Comment (sethvargo) [inline docs/deploying.md:84]: This indentation is intentional, because then nested fields (e.g. codeblocks inside a list) align correctly.

Comment (sethvargo) [reply docs/deploying.md:84]: (everywhere)

File: terraform/README.md [hunk 2]
```diff
@@ -12,16 +12,17 @@ continuous deployment.
 
 - gcloud. [Installation guide](https://cloud.google.com/sdk/install)
 
-    Note: Make sure you **unset** `GOOGLE_APPLICATION_CREDENTIALS` in your
```
Comment (sethvargo) [inline terraform/README.md:15]: This is incorrect for GHFM.

File: docs/deploying.md (hunk 1 above)
Comment (mikehelmick) [reply docs/deploying.md:84]: these are robotic changes - there is nobody on the other end to adjust

I can ask matt to turn off the markdown formatter.

**Comment by mattmoor** [comment]
it's off

**Pull request closed without merging**